content/articles/automating-windows-applications-withpython/index.md

```diff
@@ -0,0 +1,155 @@
+
+In the windows platform, different tasks can be automated using Python scripts, these tasks include launching notepad, reading and writing data in an excel spreadsheet. To approach automation with Python you will need to understand what automation is, and what python is. Automation describes a variety of technologies that reduce human intervention in the process by determination of decision criteria, sub-process, related action, and how they are related. On the other hand, Python is a high-level versatile programming language with friendly, approachable syntax, and a tone of libraries that can be utilized in task automation. 
+
+In this article, I will be demonstrating how to automate boring stuff in windows using the Pywinauto module. Pywinauto is a bundle of Python modules for automating the Microsoft Windows GUI(Graphic user interface). It allows you to send your input action to windows dialogue and control.
+
+### Table of contents
+- [Introduction](#introduction)
+- [Table of contents](#table-of-contents)
+- [Prerequisites](#prerequisites)
+- [Installation of Pywinauto](#installation-ofpywinauto)
+- [Automating notepad](#automating-notepad)
+- [Automating Excel with Pywinauto](#automating-excel-with-pywinauto)
+- [Python automation tools](#python-automation-tools)
+- [Conclusion](#conclusion)
+
+
+### Prerequisites
+To follow along, the reader must have a good knowledge of [Python](https://www.python.org).
+
+### Installation of Pywinauto
+Command to install Pwinauto :
+
+```python
+pip install pywinauto 
+```
+To manually install pywinauto install the following python packages:
+ - pyWin32
+ - comtypes
+ - six
+ - pillow (for taking screenshots)
+ - download the latest pywinauto from GitHub: https://github.com/pywinauto/pywinauto/releases
+ - unpack and run `python setup.py install`
+
+To validate yout installation run this code snippet below:
+
+```python
+from pywinauto.application import Application
+app = Application(backend="uia").start("notepad.exe")
+app.UntitledNotepad.type_keys("%FX")
+```
+This should start up your notepad application on the windows interface.
+After this installation and validation, we will jump to notepad automation which is the simple technique, followed by excel automation where we will read data from a spreadsheet and write data to the excel cells.
+
+### Automating notepad
+This example will give us an insight into how pywinauto functions by looking at functionalities like starting an application, maximizing and minimizing the application, and writing text in a notepad application. To begin, let us first import the relevant fields and launch notepad using the start method of the application imported.
+
+```python
+from pywinauto import application
+ap = application.Application ()
```

Comment:
I guess this will give a compilation error. Can you recheck?

`ap = application.Application ()`